source/components/organisms/Step/Step.js

```diff
@@ -132,9 +133,35 @@ function Step({
         formNavigation.back();
       };
 
+  const [returnScrollY, setReturnScrollY] = useState(0);
+  const scrollRef = React.useRef();
```

Comment:
For consistency, maybe drop `React.` and import `useRef` the same way as `useState`.